.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+### Expected behavior
+
+### Actual behavior
+
+### Steps to reproduce the behavior
+
+### Environment
+- rsyslog version: [e.g. 8.31.0 -- see rsyslogd -v]
+- platform: [e.g. Ubuntu 16.04]
+- for configuration questions/issues, include rsyslog.conf:
+
+*Note: rsyslog doc is available at http://www.rsyslog.com/doc*
```

Comment:
Perhaps modify the last line to something like:

> Note: rsyslog documentation is available here: `[pre-release](http://www.rsyslog.com/doc/master/)`, `[current stable release](http://www.rsyslog.com/doc/v8-stable/)` 

I offer that suggestion since the master branch is the default and it might confuse users that don't realize that it covers the upcoming version and not the current stable release version. Otherwise if you want to keep it simple, perhaps link directly to the stable version or an index of the available versions of documentation.